### PR TITLE
feat: encrypt encoded mfa settings

### PR DESCRIPTION
**Pull request by karolsojko**

### CODE REVIEW

**Review by mobitar — APPROVED** [review]

File: migrations/1629217630132-encrypt_encoded_mfa_settings.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { MigrationInterface, QueryRunner } from 'typeorm'
+import { SnCryptoNode } from '@standardnotes/sncrypto-node'
+
+export class encryptEncodedMfaSettings1629217630132 implements MigrationInterface {
```
Comment (vardan-arm) [inline migrations/1629217630132-encrypt_encoded_mfa_settings.ts:4]: Why we put random(?) numbers after class name?

**Review by vardan-arm — APPROVED** [review]

File: migrations/1629217630132-encrypt_encoded_mfa_settings.ts (hunk 1 above)
Comment (mobitar) [reply migrations/1629217630132-encrypt_encoded_mfa_settings.ts:4]: They are timestamps :)

**Review by antsgar — APPROVED** [review]

File: migrations/1629217630132-encrypt_encoded_mfa_settings.ts (hunk 1 above)
Comment (vardan-arm) [reply migrations/1629217630132-encrypt_encoded_mfa_settings.ts:4]: I see, but why? :) I believe it's not done to disguise class names, but can't understand the reasoning behind it.

Comment (mobitar) [reply migrations/1629217630132-encrypt_encoded_mfa_settings.ts:4]: Common in migrations, to order them. Otherwise you have a bunch of unordered classes.

Comment (vardan-arm) [reply migrations/1629217630132-encrypt_encoded_mfa_settings.ts:4]: Ah, got it, thanks :)

**Pull request merged**